frontends/mit-learn/src/page-components/ResourceCard/ResourceCard.tsx

```diff
@@ -124,6 +134,7 @@ const ResourceListCard: React.FC<ResourceListCardProps> = ({
   condensed,
   ...props
 }) => {
+  const loc = useLocation()
```

Comment:
It seems like there is a lot of redundancy between ResourceCard and ResourceListCard that could potentially be abstracted/consolidated.